corehq/apps/geospatial/static/geospatial/js/geo_config.js

```diff
@@ -46,6 +50,32 @@ hqDefine("geospatial/js/geo_config", [
             return self.selectedGroupMethod() === minMaxGroupingName;
         });
 
+        self.selectedAlgorithm = ko.observable();
+        self.plaintext_api_token = ko.observable(data.plaintext_api_token);
+
+        self.captureApiToken = ko.computed(function () {
+            return self.selectedAlgorithm() === ROAD_NETWORK_ALGORITHM;
+        });
+
+        self.validateApiToken = function () {
+            const url = "https://api.mapbox.com/directions-matrix/v1/mapbox/driving/-122.42,37.78;-122.45,37.91;-122.48,37.73";
```

Comment:
just confirming that mapbox does not provide an endpoint just to validate the token?

Reply:
@mkangia No, I'm not aware of any such endpoint.

Reply:
Cool, thanks @Charl1996